### DISCUSSION TITLE
AES Key Wrap implementation

### DISCUSSION THREAD

**Discussion by mirceanis** [paulmillr/noble-ciphers#18] [Ideas]
Is there any plan to add support for AES Key Wrap (https://datatracker.ietf.org/doc/html/rfc3394#section-2.2.1) ?
I'd like to be able to use this library for `A256KW`

**Comment by paulmillr** [comment]
maybe. is it popular?

**Reply by mirceanis** [reply]
I believe it is.
It's available through webcrypto and it is one of the recommended ones in JSON Web Algorithms.
